LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
```

Comment:
Remove the license information. The proprietary license will be added when ready.